ProjectEureka/project-eureka-flutter
# PE-73 | Profile Service and UI Changes to Answers

What I Did:

- Created Answers model.
- Created UI for profile answers list which is mostly based on 'eureka_list_view'.
- Connected user information (avatar, first name, last name, email, and rating) with backend
- Connected Answers and Questions according to the current user ID.
- Added "no results" message in case when no answers or questions were associated with the user id (profile's GET request returned empty lists)
- Combined previously separated GET operations for profile's questions and answers.

How to Test:

- Make sure to have environmental variables file in the root folder and run 'flutter pub get'.
- Sign in with Johnny Cash account, which already has information to demo profile page. Username: "johnnycash123@email.com", Password: "Jc12345!"
- Google Account would work too, but currently it is likely that your google account is not connected to User's table, so profile info may not load.
- Go to Profile screen. All tabs must display either lists or a message " have not posted ... yet".
- Follow the lines with "PE-73 testing:" comments (use search) in profile_screen.dart file to test different cases. After applying changes, you may sometimes need to reload page from side menu.
- Check if the loading indicator works properly in all cases.
